### DISCUSSION TITLE
输入法插件卸载后，输入法框架的自动刷新，去掉不完整的输入法

### DISCUSSION THREAD

**Discussion by xingwozhonghua126** [fcitx/fcitx5#354] [Ideas]
输入法插件卸载后，输入法框架的自动刷新，去掉不完整的输入法。

卸载输入法时通过，软件脚本发送fcitx-remote u,框架接受到信号后，重新读取输入法配置文件。若当然已加载的输入法插件不正确，将插件状态改为已卸载（使能/不使能/已卸载）。即在待添加页面和已添加页面都不存在的输入法。

**Comment by wengxt** [comment]
卸载是很危险的，并不打算支持动态卸载。如果无法加载的情况已经会显示（不可用）了。
